### PR TITLE
Tools of caffe_convert : support 1X7 convolution kernel and pad_w,pad_h params

### PR DESCRIPTION
**Pull request by likelyzhao**
hi all 
   the original caffe_convert code dose not support the caffe convolution layer with kernal size like 1X7 and deal with the param pad_w and pad_h  params, this pr is fix this issue

### CODE REVIEW

**Comment by mli** [comment]
could you please merge the master and fix the possible lint errors? 

#6264 enabled lint for the caffe converter

**Comment by likelyzhao** [comment]
merge and fixed @mli

File: tools/caffe_converter/convert_symbol.py [hunk 1]
```diff
@@ -25,32 +25,60 @@ def _get_input(proto):
...
+            pad = param.pad[0]
+            param_string += ", pad=(%d, %d)" % (pad, pad)
+        else:
+            if isinstance(param.pad_w, int):
+                pad_w = param.pad_w
+            if isinstance(param.pad_h, int):
+                pad_h = param.pad_h
+            param_string += ", pad=(%d, %d)" % (pad_h, pad_w)
+
+    kernel_w = ''
```
Comment (mli) [inline tools/caffe_converter/convert_symbol.py]: is it necessary?

File: tools/caffe_converter/convert_symbol.py [hunk 2]
```diff
@@ -25,32 +25,60 @@ def _get_input(proto):
...
-        pad = 0 if len(param.pad) == 0 else param.pad[0]
+        if len(param.kernel_size) > 0:
+            kernel_size = param.kernel_size[0]
+            param_string += ", kernel=(%d,%d)" % (kernel_size, kernel_size)
+        else:
+            if isinstance(param.kernel_w, int):
+                kernel_w = param.kernel_w
+            if isinstance(param.kernel_h, int):
+                kernel_h = param.kernel_h
+            param_string += ", kernel=(%d,%d)" % (kernel_h, kernel_w)
```
Comment (mli) [inline tools/caffe_converter/convert_symbol.py]: should we change to `assert isinstance(param.kernel_w, int)`

File: tools/caffe_converter/convert_symbol.py [hunk 3]
```diff
@@ -25,32 +25,60 @@ def _get_input(proto):
...
         stride = 1 if len(param.stride) == 0 else param.stride[0]
-    kernel_size = ''
-    if isinstance(param.kernel_size, int):
-        kernel_size = param.kernel_size
-    else:
-        kernel_size = param.kernel_size[0]
+
+    param_string += ", stride=(%d,%d)" % (stride, stride)
+
+
```
Comment (mli) [inline tools/caffe_converter/convert_symbol.py]: remove duplicated empty lines

File: tools/caffe_converter/convert_symbol.py (hunk 2 above)
Comment (likelyzhao) [inline tools/caffe_converter/convert_symbol.py]: it seem assert is better than if

**Comment by mli** [comment]
Many thanks, merged.

**Pull request merged**